libs/java/dal/db/dal-db/src/main/java/ca/bc/gov/tno/dal/db/entities/MediaType.java

```diff
@@ -171,8 +171,8 @@ public void setEnabled(boolean enabled) {
   /**
    * @param dataSources the dataSources to set
    */
-  public void setDataSources(List<DataSource> dataSources) {
-    this.dataSources = dataSources;
+  public void setDataSources(List<DataSource> data_sources) {
```

Comment:
Incorrect naming convention, change to `dataSources`

Reply:
Fixed it.